test/DocumentFormat.OpenXml.Tests/Spreadsheet/CellValueTests.cs

```diff
@@ -74,6 +76,26 @@ public void CellDoubleTest(double num)
             Assert.Equal(num, result);
         }
 
+        [Fact]
+        public void CellDoubleCultureTest()
```

Comment:
what about for int/dates/etc? May not be as big of a deal, but with exponential support added in, it may

Reply:
Dates definitely could be included. Not sure about int but I'll take some time to check them out.

Reply:
I checked dates and they don't work the same way as decimal and double numbers. I tested a couple different dates but Excel seems to pay attention to the incoming format, unlike decimal and double.